onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -61,7 +61,17 @@
         "^test_mul_uint8_cpu",
         "^test_add_uint8_cpu",
         "^test_div_uint8_cpu",
-        "^test_batchnorm_*"
+        //GPU failures
+        "^test_batchnorm_epsilon_training_mode_cuda",
+        "^test_batchnorm_example_training_mode_cuda",
+        "^test_convtranspose_autopad_same_cpu",
+        "^test_sub_uint8_cuda",
```

Comment:
all the tests starting line 68 to 74 are not supported on both cuda and cpu so changing this to "^test_sub_uint8_*" should work too